__tests__/app.test.js

```diff
@@ -87,6 +87,55 @@ describe('GET /api/articles/:article_id', () => {
     
 });
 
+describe('GET /api/articles/:article_id/comments', () => {
+  test('Should respond with an array of comments for the given article_id of which each comment should have the following properties: comment_id, votes, created_at, author, body, article_id', () => {
+    return request(app)
+      .get('/api/articles/1/comments')
+      .expect(200)
+      .then((response) => {
+        const comments = response.body;
+        console.log(comments, "comments");
+        expect(Array.isArray(comments)).toBe(true)
+
+        comments.forEach((comment) => {
```

Comment:
we should by also adding an assertion for checking the typeof the property values as well as checking that it has the right keys.